vars/main.yml

```diff
@@ -0,0 +1,11 @@
+nginx_access_log: "/var/log/nginx/mattermost_access.log"
+nginx_error_log: "/var/log/nginx/mattermost_error.log"
+postgres_initdb: "/usr/lib/postgresql/9.5/bin/initdb"
```

Comment:
make version configurable
also, it is another one as in the service file

Reply:
maybe find the path using `which` like in xen_vman.
The way I understand it, this will change when I update postgres.

Reply:
Just WTF